java/src/jmri/jmrit/throttle/ThrottleBundle_de.properties

```diff
@@ -61,6 +61,7 @@ ButtonProgram = Programmieren
 AddressInUse = Adresse bereits einem anderen Fahrregler zugewiesen.
 
 # ControlPanelPropertyEditor
+ControlPanelProperties =  Eigenschaften
```

Comment:
Fixed (chose to remove line 158 instead of line 64 as it fits better under the comment here)